README.md

```diff
@@ -22,7 +22,7 @@ Refer to the [core usage guidelines](https://github.com/pattern-lab/patternlab-n
 
 ### Installation
 
-As of Pattern Lab Node 3.0.0, installation of [Editions](http://patternlab.io/docs/advanced-ecosystem-overview.html) is accomplished via the command line interface.
+As of Pattern Lab Node 3.0.0, installation of [Editions](http://patternlab.io/docs/advanced-ecosystem-overview.html) is accomplished via the [command line interface](https://github.com/pattern-lab/patternlab-node/tree/master/packages/cli) which needs to be [installed](https://github.com/pattern-lab/patternlab-node/tree/master/packages/cli#installation) into the `new-project` folder mentioned below)
```

Comment:
👋 @digiltd 

I am trying to better understand how this tripped you up, since the installation instructions directly below this utilize `npx @pattern-lab/cli` to install the CLI directly. Both editions you can install include the CLI as a dependency, so it should be available post-generation

Reply:
I am pretty sure when I just tried `npm init -y && npx @pattern-lab/cli -c patternlab init` it didn't work. 

Though I just tried again with a new directory and it worked first time with no problems. I have no idea what caused the error before. I definitely used the same command each time (I just checked my `bck-i-search` to make sure)

I will close the PR as it was probably something at my end. But thanks for checking in :)